lib/ansible/modules/packaging/os/portage.py

```diff
@@ -156,6 +156,8 @@
   jobs:
     description:
       - Specifies the number of packages to build simultaneously.
+      - Since version 2.6: Value of 0 resets any previously added
```

Comment:
you need to quote : as it is a 'yaml' delimiter

Reply:
Oops. Btw why ain't yaml blocks used here?